code/common/datastripe.q

```diff
@@ -22,3 +22,28 @@ deletetablebefore:{![x;enlist (<;y;z);0b;0#`]}
 
 //Function to check if a segid is defined if datastriping is on
 checksegid:{if[datastripe;if[not (`segid in key .proc.params);.lg.e[`init;"Datastriping is turned on however no segment id has been defined for this process"]]]}
+
+// function to filter replayed tables with where clause from striping.json
+applyfilters:{[filtertab;td]
+  if[count filtertab;
+   .lg.o[`subscribe;"filtering table(s) ", (", " sv string filtertab), " started at: ",string .z.P];
+   set'[filtertab; filtertable[;td] each filtertab];
+   .lg.o[`subscribe;"finished filtering table(s) ", (", " sv string filtertab), " at: ",string .z.P];
```

Comment:
The indenting here looks off, the end square bracket ends up less endented than the if body, it might just be preference but I think something like below is more readable

```suggestion
    .lg.o[`subscribe;"filtering table(s) ", (", " sv string filtertab), " started at: ",string .z.P];
    set'[filtertab; filtertable[;td] each filtertab];
    .lg.o[`subscribe;"finished filtering table(s) ", (", " sv string filtertab), " at: ",string .z.P];
```